contracts/contracts/chains/arbitrum/ArbitrumMessengers.sol

```diff
@@ -75,6 +75,8 @@ contract ArbitrumL1Messenger is IMessenger, RestrictedCalls {
             0,
             message
         );
+
+        this.withdraw();
```

Comment:
is there a reason why you chose to make an external function call to withdraw?
Why not changing the visibility of withdraw to `public`?

see https://docs.soliditylang.org/en/v0.4.21/contracts.html#visibility-and-getters

Reply:
It was just lack of knowledge tbh. Will change it to a public one